Lib/test/test_exceptions.py

```diff
@@ -2596,5 +2596,50 @@ def after_with():
                 1/0
         self.lineno_after_raise(after_with, 1, 1)
 
+    def test_repr(self):
```

Comment:
Those tests are actually in the wrong place as they don't have anything to do with PEP-626 I think. We have test somewhere that already test `repr` of exception types but I don't know where. Ideally, I would also rename those test methods as `test_import_error_repr` instead of extending an existing test that is very long.

Reply:
oh nice catch - will move them under the `ImportErrorTest` Class